src/renderer/src/helpers/StringHelper.ts

```diff
@@ -6,6 +6,14 @@ export class StringHelper {
     return str
   }
 
+  static truncateStringStart(str: string, maxLength: number) {
+    if (str.length > maxLength) {
+      const half = maxLength / 2
+      return '...' + str.substring(str.length - half)
```

Comment:
Use a template string.

Reply:
it's harder to read with a template string